src/datasources/targeted-messaging/outreach-file-processor.spec.ts

```diff
@@ -138,13 +139,33 @@ describe('OutreachFileProcessor', () => {
     );
   });
 
+  it('should not attempt to process Outreach data files when tageting all safes', async () => {
+    const created = await targetedMessagingDatasource.createOutreach(
+      createOutreachDtoBuilder()
+        .with('sourceFile', null)
+        .with('sourceFileChecksum', null)
+        .with('targetAll', true)
+        .build(),
+    );
+
+    await fileProcessor.onModuleInit();
+
+    expect(
+      await targetedMessagingDatasource.getUnprocessedOutreaches(),
+    ).toHaveLength(1);
+    expect(mockLoggingService.info).toHaveBeenCalledWith(
+      `[Outreach ${created.id}] Targeting all safes. No file to process`,
+    );
+  });
+
   it('should not process Outreach data files with bad checksum', async () => {
     const sourceFile = path.resolve(baseDir, fileName);
     const expectedChecksum = faker.string.alphanumeric();
     const created = await targetedMessagingDatasource.createOutreach(
       createOutreachDtoBuilder()
         .with('sourceFile', sourceFile)
         .with('sourceFileChecksum', expectedChecksum)
+        .with('targetAll', false)
```

Comment:
Question: why is it added? Does it make a difference if it's false or true here?

Reply:
if targetAll is will not try to process the file, so it wont check the checksum